### PR TITLE
CLOUD-2194: Upgrade build software componenten in jenkins-ecs-agent

### PR DESCRIPTION
**Pull request by r-kok**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -134,13 +127,12 @@ ENV N_PREFIX=${HOME}/n
...
 RUN curl -sL http://git.io/n-install -o /tmp/n-install.sh \
  && chmod +x /tmp/n-install.sh \
- && /tmp/n-install.sh -y 6.9.0 6.3.1 4.1.2 8.11.1 9.11.1 \
+ && /tmp/n-install.sh -y 10.15.3 \
  && rm /tmp/n-install.sh
 
 
 ### Install NodeJS global stuff
-RUN npm install -g bower@1.3.12 grunt-cli@0.1.13 @angular/cli@1.0.0 \
- && npm cache clean
```
Comment (melvyndekort) [inline Dockerfile]: Can you check whether the image grows from removing this step?
If not we shouldn't remove it or replace it with something similiar.

File: Dockerfile [hunk 2]
```diff
@@ -134,13 +127,12 @@ ENV N_PREFIX=${HOME}/n
...
+RUN npm install -g bower@1.3.12 grunt-cli@0.1.13 @angular/cli@1.0.0
```
Comment (snorberhuis) [inline Dockerfile]: Is it possible to update bower? This bower version does not work correctly as it uses an old url: http://bower.herokuapp.com/packages/angular-scenario

Comment (snorberhuis) [reply Dockerfile]: I need this for an issue I am working on.

Comment (r-kok) [reply Dockerfile]: alles kan maar maak daar een svp apart issue van, in plaats van dit PR te hijacken

**Pull request merged**